.azure-pipelines.yml

```
@@ -200,6 +216,22 @@ jobs:
       conda list
     displayName: List installed packages
 
+  # Cache the ${HOME}/.gmt directory, for docs and testing
+  - task: Cache@2
+    inputs:
+      key: cachedata | 20200519
+      path: $(HOME)/.gmt
```

Use **HOMEPATH** on Windows.
```suggestion
      path: $(HOMEPATH)/.gmt
```